flink-yarn-tests/src/test/java/org/apache/flink/yarn/util/FakeClusterClient.java

```diff
@@ -93,4 +93,9 @@ public String stopWithSavepoint(JobID jobId, boolean advanceToEndOfEventTime, @N
 	public Map<String, OptionalFailure<Object>> getAccumulators(JobID jobID, ClassLoader loader) {
 		return Collections.emptyMap();
 	}
+
+	@Override
+	public void close() throws Exception {
```

Comment:
Wried. The latest CI complaint with

```
/home/travis/build/flink-ci/flink/flink-yarn-tests/src/test/java/org/apache/flink/yarn/util/FakeClusterClient.java:[43,8] org.apache.flink.yarn.util.FakeClusterClient is not abstract and does not override abstract method close() in java.lang.AutoCloseable
```

but actually we implement it.

Reply:
Let's see what travis says now.